### PR TITLE
Claim winnings instead of burning shares when a market is resolved.

### LINKED ISSUE #639: When removing liquidity for a resolved market, use the claimWinnings method
This is the contract work to fix the bug from #627.

When a market is resolved, the AMMFactory needs to claimWinnings not burnShares.

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/smart/contracts/turbo/AMMFactory.sol [hunk 1]
```diff
@@ -218,7 +218,11 @@ contract AMMFactory is BNum {
...
 
-        _collateralOut = _marketFactory.burnShares(_marketId, _setsToSell, _collateralRecipient);
+        if (_marketFactory.isMarketResolved(_marketId)) {
+            _collateralOut = _marketFactory.claimWinnings(_marketId, _collateralRecipient);
+        } else {
+            _collateralOut = _marketFactory.burnShares(_marketId, _setsToSell, _collateralRecipient);
+        }
         require(_collateralOut > _minCollateralOut, "Amount of collateral returned too low.");
 
         // Transfer the remaining shares back to _collateralRecipient.
```
Comment (justinbarry) [inline packages/smart/contracts/turbo/AMMFactory.sol:228]: @robert-e-davidson3 Should we still transfer out dead shares?

**Review by justinbarry — CHANGES_REQUESTED** [review]
One comment

File: packages/smart/contracts/turbo/AMMFactory.sol (hunk 1 above)
Comment (robert-e-davidson3) [reply packages/smart/contracts/turbo/AMMFactory.sol:228]: I don't think it matters.

Comment (justinbarry) [reply packages/smart/contracts/turbo/AMMFactory.sol:228]: Cool.

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**